validation-test/stdlib/XCTest.swift

```diff
@@ -252,52 +252,118 @@ XCTestTestSuite.test("XCTAssertThrowsError") {
 
 }
 
-XCTestTestSuite.test("XCTAsserts with throwing expressions") {
-    class ErrorTestCase: XCTestCase {
-        var doThrow = true
-        var errorCode = 42
-        
-        dynamic func throwSomething() throws -> String {
-            if doThrow {
-                throw NSError(domain: "MyDomain", code: errorCode, userInfo: nil)
-            }
-            return "Hello"
-        }
-        
-        dynamic func test_withThrowing() {
-            XCTAssertEqual(try throwSomething(), "Hello")
-        }
+XCTestTestSuite.test("XCTAssertNoThrow") {
+  class ErrorTestCase: XCTestCase {
+    var doThrow = true
+    var errorCode = 42
+
+    dynamic func throwSomething() throws {
+      if doThrow {
+        throw NSError(domain: "MyDomain", code: errorCode, userInfo: nil)
+      }
     }
-    
-    // Try success case
-    do {
-        let testCase = ErrorTestCase(selector: #selector(ErrorTestCase.test_withThrowing))
-        testCase.doThrow = false
-        execute(testCase.run)
-        let testRun = testCase.testRun!
-        
-        expectEqual(1, testRun.testCaseCount)
-        expectEqual(1, testRun.executionCount)
-        expectEqual(0, testRun.failureCount)
-        expectEqual(0, testRun.unexpectedExceptionCount)
-        expectEqual(0, testRun.totalFailureCount)
-        expectTrue(testRun.hasSucceeded)
+
+    dynamic func test_throws() {
+
+      XCTAssertNoThrow(try throwSomething()) {
```

Comment:
This block is a copy-and-paste error.  This API does not have an error callback.